packages/grid_client/tests/modules/applications/taiga.test.ts

```diff
@@ -53,6 +53,7 @@ test("TC2691 - Applications: Deploy Taiga", async () => {
 
   //Test Data
   const name = "gw" + generateString(10).toLowerCase();
+  const subdomain = name;
```

Comment:
why we do this, we can use the name; or add prefix `solutionCode`+`twin_id` to the name as we are doing in playground